examples/simple_muscles.py

```diff
@@ -0,0 +1,87 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import sympy as sm
+import sympy.physics.mechanics as me
+from sympy.physics.mechanics.pathway import LinearPathway
+
+from biomechanics import (FirstOrderActivationDeGroote2016,
+                          MusculotendonDeGroote2016)
+from biomechanics.plot import plot_config, plot_traj
+
+q, u = me.dynamicsymbols('q, u')
+m, g = sm.symbols('m, g')
+F_M_max, l_M_opt, l_T_slack = sm.symbols('F_M_max, l_M_opt, l_T_slack')
+v_M_max, alpha_opt, beta = sm.symbols('v_M_max, alpha_opt, beta')
+
+N = me.ReferenceFrame('N')
+O, P = sm.symbols('O, P', cls=me.Point)
+
+P.set_pos(O, q*N.x)
+O.set_vel(N, 0)
+P.set_vel(N, u*N.x)
+
+gravity = me.Force(P, m*g*N.x)
+
+muscle_pathway = LinearPathway(O, P)
+muscle_activation = FirstOrderActivationDeGroote2016('big_dog').with_default_constants('muscle')
+muscle = MusculotendonDeGroote2016(
+    'muscle',
+    muscle_pathway,
+    activation_dynamics=muscle_activation,
+    tendon_slack_length=l_T_slack,
+    peak_isometric_force=F_M_max,
+    optimal_fiber_length=l_M_opt,
+    maximal_fiber_velocity=v_M_max,
+    optimal_pennation_angle=alpha_opt,
+    fiber_damping_coefficient=beta,
+)
+
+block = me.Particle('block', P, m)
+
+kane = me.KanesMethod(N, (q,), (u,), kd_eqs=(u - q.diff(),))
+kane.kanes_equations((block,), (muscle.to_loads() + [gravity]))
+
+a = muscle.activation_dynamics.state_variables[0]
+e = muscle.activation_dynamics.control_variables[0]
+
+dqdt = u
+dudt = kane.forcing[0]/m
+dadt = list(muscle.activation_dynamics.state_equations.values())[0]
+
+state = [q, u, a]
+inputs = [e]
+constants = [m, g, F_M_max, l_M_opt, l_T_slack, v_M_max, alpha_opt, beta]
+
+eval_eom = sm.lambdify((state, inputs, constants), (dqdt, dudt, dadt))
+
+x_vals = np.array([
+    0.0,
+    0.0,
+    0.0,
+])
+
+r_vals = np.array([
+    0.0,
+])
+
+p_vals = np.array([
+    1.0,  # m [kg]
+    9.81,  # g
+    500.0,
+    0.18,
+    0.17,
+    10.0,
+    0.0,
+    0.1,
+])
+
+print(eval_eom(x_vals, r_vals, p_vals))
```

Comment:
So udot evaluates to nan instead of g. I'm not sure why.

If I give q and initial value of some length, udot is also not g but some large number.

It seems the muscle produces force even it excitation and activation are both zero:

```
In [2]: kane.forcing[0].doit().xreplace({a: 0, e: 0})
Out[2]: F_M_max*(-beta*(-l_T_slack + sqrt(q(t)**2))*sqrt(q(t)**2)*u(t)/(v_M_max*sqrt(l_M_opt**2*sin(alpha_opt)**2 + (-l_T_slack + sqrt(q(t)**2))**2)*q(t)) + 2.37439874427164e-5*exp(6.66666666666667*sqrt(l_M_opt**2*sin(alpha_opt)**2 + (-l_T_slack + sqrt(q(t)**2))**2)/l_M_opt) - 0.0186573603637741)*q(t)/sqrt(q(t)**2) + g*m

In [3]: kane.forcing[0].doit().xreplace({q: 0, a: 0, e: 0})
Out[3]: nan
```

Reply:
This is as expected. We have a rigid tendon musculotendon unit here, which is essentially a nonlinear spring, linear damper, and force producing unit in parallel. If parameterised sensibly it will be force producing at most lengths under no excitation/activation because of the spring part, even though this force will be very small until it's length becomes a bit longer than the tendon slack length.

Reply:
I think the `nan` is popping out because the attachments of the musculotendon are separated by `q(t)`, i.e. the musculotendon length is `q(t)`, so when you replace this with `0` the musculotendon has no length which doesn't really make physical sense. 

The force equation is put together in a way that assumes musculotendon length is always positive and greater than the tendon slack length. This is because we need to work out the muscle fiber length in order to evaluate the characteristic curves at its current state to get the force. So we do `l_M = l_MT - l_T` (or muscle fiber length is musculotendon length minus tendon length), with some extra complications taking account of pennation angle in there too.

Reply:
Ok, that helps and we can explain it in the documentation.